BookingsApi/BookingsApi.Contract/Requests/GetHearingRequest.cs

```diff
@@ -12,6 +12,9 @@ public class GetHearingRequest
 
         [JsonProperty("types")]
         public List<int> Types { get; set; }
+        
+        [JsonProperty("users")]
+        public List<Guid> Users { get; set; }
```

Comment:
why are json property attributes set on request models and not response models?

Reply:
those properties were always there.... if you would like to change this you should rise a new ticket for refactoring